ERROR:seismic_client(h0-b1):Error in SeismicClient.run() can't recover: (9, 'Bad file descriptor')
